### DISCUSSION TITLE
TypeError: float() argument must be a string or a real number, not 'dict'

### DISCUSSION THREAD

**Discussion by yerin723** [mindee/doctr#1302] [Q&A]
In terminal
$ python train_pytorch.py ​​'C:/Users/PIAI/Desktop/test' 'C:/Users/PIAI/Desktop/val' db_resnet50
I ran it and
The following error occurred:

PIAI@DESKTOP-83330NQ MINGW64 ~/Desktop/ocrproject/doctr/references/detection (main)
$ python train_pytorch.py 'C:/Users/PIAI/Desktop/test' 'C:/Users/PIAI/Desktop/val' db_resnet50
C:\Users\PIAI\Desktop\ocrproject\venv\Lib\site-packages\torchvision\datapoints\__init__.py:12: UserWarning: The torchvision.datapoints and torchvision.transforms.v2 namespaces are still Beta. While we do not expect major breaking changes, some APIs may still change according to user feedback. Please submit any feedback you may have in this issue: https://github.com/pytorch/vision/issues/6753, and you can also check out https://github.com/pytorch/vision/issues/7319 to learn more about the APIs that we suspect might involve future changes. You can silence this warning by calling torchvision.disable_beta_transforms_warning().
  warnings.warn(_BETA_TRANSFORMS_WARNING)
C:\Users\PIAI\Desktop\ocrproject\venv\Lib\site-packages\torchvision\transforms\v2\__init__.py:54: UserWarning: The torchvision.datapoints and torchvision.transforms.v2 namespaces are still Beta. While we do not expect major breaking changes, some APIs may still change according to user feedback. Please submit any feedback you may have in this issue: https://github.com/pytorch/vision/issues/6753, and you can also check out https://github.com/pytorch/vision/issues/7319 to learn more about the APIs that we suspect might involve future changes. You can silence this warning by calling torchvision.disable_beta_transforms_warning().
  warnings.warn(_BETA_TRANSFORMS_WARNING)
Namespace(train_path='C:/Users/PIAI/Desktop/test', val_path='C:/Users/PIAI/Desktop/val', arch='db_resnet50', name=None, epochs=10, batch_size=2, device=None, input_size=1024, lr=0.001, weight_decay=0, workers=None, resume=None, test_only=False, freeze_backbone=False, show_samples=False, wb=False, push_to_hub=False, pretrained=False, rotation=False, eval_straight=False, sched='cosine', amp=False, find_lr=False)
Traceback (most recent call last):
  File "C:\Users\PIAI\Desktop\ocrproject\doctr\references\detection\train_pytorch.py", line 456, in <module>
    main(args)
  File "C:\Users\PIAI\Desktop\ocrproject\doctr\references\detection\train_pytorch.py", line 184, in main
    val_set = DetectionDataset(
              ^^^^^^^^^^^^^^^^^
  File "C:\Users\PIAI\Desktop\ocrproject\venv\Lib\site-packages\doctr\datasets\detection.py", line 61, in __init__
    polygons: np.ndarray = np.asarray(label["polygons"], dtype=np_dtype)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'dict'

Tell me the reason

**Comment by felixT2K** [comment]
Hi @yerin723 👋🏼,

Looks like your dataset does not follow the required data format (ref.: https://github.com/mindee/doctr/tree/main/references/detection#data-format)

Would it be possible to share an entry from your labels.json ?
